test/web-platform-tests/to-run.yaml

```diff
@@ -53,25 +53,25 @@ cssom-getBoxQuads-001.html: [fail, Not implemented]
 cssom-getClientRects-002.html: [fail, Unknown]
 cssom-view-img-attributes-001.html: [fail, Unknown]
 dom-element-scroll.html: [fail, Scroll methods not implemented]
-elementFromPoint-001.html: [fail, Not implemented]
-elementFromPoint-002.html: [fail, Not implemented]
-elementFromPoint-003.html: [fail, Not implemented]
-elementFromPoint-dynamic-anon-box.html: [fail, Not implemented]
-elementFromPoint-mixed-font-sizes.html: [fail, Not implemented]
-elementFromPoint.html: [fail, Not implemented]
+elementFromPoint**: [timeout, Not implemented]
```

Comment:
Is it possible to keep these as fail? Fine if not.

Reply:
Sure.